Minio/DataModel/ObjectOperationsArgs.cs

```diff
@@ -400,4 +402,81 @@ public RemoveIncompleteUploadArgs()
             this.RequestMethod = Method.DELETE;
         }
     }
-}
+
+    public class GetObjectArgs : ObjectQueryArgs<GetObjectArgs>
+    {
+        internal Action<Stream> CallBack { get; private set; }
+        internal long ObjectOffset { get; private set; }
+        internal long ObjectLength { get; private set; }
+        internal string FileName { get; private set; }
+        internal bool OffsetLengthSet { get; set; }
+
+        public GetObjectArgs()
+        {
+            this.RequestMethod = Method.GET;
+            this.OffsetLengthSet = false;
+        }
+
+        public override void Validate()
+        {
+            base.Validate();
+            if (this.CallBack == null)
+            {
+                throw new MinioException("CallBack method not set of GetObject operation.");
+            }
+            if (OffsetLengthSet)
+            {
+                if (this.ObjectOffset < 0)
+                {
+                    throw new ArgumentException("Offset should be zero or greater", nameof(this.ObjectOffset));
+                }
+
+                if (this.ObjectLength < 0)
+                {
+                    throw new ArgumentException("Length should be greater than zero", nameof(this.ObjectLength));
```

Comment:
message should be greater than or equal to

Reply:
Changed the message & the condition.